### PR TITLE
Feat/camera integration

### PR DESCRIPTION
**Pull request by aladin6060**

### CODE REVIEW

**Comment by aladin6060** [comment]
Somehow the ui tests only fail on cirrus but all pass locally. Something of CameraPreviewTest messes with cirrus

**Review by gaetschwartz — APPROVED** [review]
LGTM, we should try to see if it's possible to reach a better coverage for this, but seems hard to test more as of now.

**Comment by aladin6060** [comment]
Yes, I agree that more tests are needed. I think it's better to do more tests after the whole ml pipeline is integrated. Otherwise, some of the implemented tests can't be used.

File: app/build.gradle [hunk 1]
```diff
@@ -104,8 +106,18 @@ dependencies {
 
     testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.0-native-mt'
     androidTestImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.0-native-mt'
-
     testImplementation 'org.mockito:mockito-core:4.4.0'
+
+    //Declare the dependency for camera related functions
+    def camerax_version = "1.1.0-beta01"
```
Comment (bhvrgav) [inline app/build.gradle:112]: Could we use the newer version of the camera that is available?

**Review by panadax — APPROVED** [review]
LGTM, I like it!

**Review by adam-narozniak — COMMENTED** [review]
All the code is clear.
Remove, just please the .idea folder (I think that I committed it on the first commit without proper gitignore and now it still on some branches but it's removed on main)

**Pull request merged**